Source/Scripts/MainClass.cs

```diff
@@ -4,10 +4,12 @@
 
 namespace IntegrationTests;
 
-public class Program {
+public class MainClass {
   public static Task Main(string[] args) {
     var root = new RootCommand("Various scripts that help develop Dafny");
     root.AddCommand(UpdateTests.GetCommand());
+    root.AddCommand(ParsedAstGenerator.GetCommand());
+    root.AddCommand(DeserializerGenerator.GetCommand());
```

Comment:
Worth adding top-level Makefile targets for these? Especially since then you can provide the correct default arguments for updating the parsed AST.